README.md

```diff
@@ -41,6 +41,25 @@ The following bounties are available for anyone to pick up:
 
 <br>
 
+# Bounty Submission
+
+## Outside of [XRPLF](https://github.com/XRPLF)
+
+To submit your bounty work(finished or work in progress) for review, you will share your submission by opening a PR to new open source repo(s) under the [XRPL Bounties github organization](https://github.com/XRPLBounties).
+
+This will make it easier for reviewers and submitters to collaborate (comment on specific line of codes and request changes). It will also increase the publicity of your work, as the community will be able to find everything under XRPL Bounties.
```

Comment:
```suggestion
This will allow reviewers to directly comment on specific lines of code, clearly indicate when a bounty is accepted, and help increase the publicity for your work as the community will be able to find everything under the [XRPL Bounties GitHub organization](https://github.com/XRPLBounties).
```